Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -197,39 +293,13 @@
     factions:
     - PetsNT
   - type: Tool
-    speed: 3
+    speed: 6
```

Comment:
```suggestion
    speed: 3
```

Reply:
this is a value only applicable to Friend shaped and Patriarch, my admeme xenos, I was going to test out a different value for the prying on them. 

I disagree with your opinion in both cases.